src/Universum/String/Conversion.hs

```diff
@@ -180,7 +180,21 @@ explanation in the comment below.
 
 {- [Note toString-toText-rewritting]
 
-We can do even better if take rules defined in 'Data.Text' into account.
+We can do even better than above if take rules defined in 'Data.Text' into
+account.
+
+Note ON MAINTENANCE: whenever you need to update the used version of @text@ package,
+you have to check whether the comment below is still valid.
+However, you can cut down by looking at the diff between versions of @text@, and
```

Comment:
I would also include a link like https://github.com/haskell/text/compare/1.2.4.1...release-1.2.5.0 that the reader can adapt for their specific case (by updating versions in it).

Reply:
That would be neat, but I'm not sure which exact policy do they have (e.g. there is no `1.2.5.0` tag for some reason yet). This link I found in one of the issues accidentally.

Reply:
At least we can include https://github.com/haskell/text link, it will take less steps to review the diff from here and I guess this link is unlikely to change (source code is unlikely to be moved anywhere).